src/batch-client/index.ts

```diff
@@ -512,6 +515,14 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public sendInviteReply = (requestOptions: InviteReplyInput) =>
+		this.jsonRequest({
+			name: 'SendInviteReply',
+			body: {
+				...denormalize(InviteReply)(requestOptions)
```

Comment:
We can avoid the extra object clone here:

```js
body: denormalize(InviteReply)(requestOptions)
```

It is written like that in a few places in the code, but those should eventually be changed to not do an extra clone also.